app/javascript/react/screens/App/Plan/Plan.js

```diff
@@ -155,7 +154,7 @@ class Plan extends React.Component {
         <Toolbar>
           <Breadcrumb.Item href="/dashboard/maintab?tab=compute">{__('Compute')}</Breadcrumb.Item>
           <li>
-            <Link to="/">{__('Migration')}</Link>
+            <a href="/migration/overview">{__('Migration')}</a>
```

Comment:
Changing our `<Link />` components to anchor tags will unfortunately disrupt some existing behavior.  When clicking on the breadcrumb to navigate back to Overview, the selected Plan view should be preserved

i.e.,
1.  User is viewing completed plans
2.  User clicks on a plan to view its details
3.  User navigates back to overview
4.  User should still be viewing completed plans

Is possible to keep the `react-router-dom` component?  Or will that cause issues?

Reply:
(The short story is ... if there is no full page reload between plan detail screen and overview screen, we can't keep track of where the user is on the backend.)

Reply:
Ok, that is reasonable.  After discussing with @mturley and @vconzola, we understand that this is necessary for us to be able to move forward.

@mzazrivec let's discuss this during tomorrow's sync with the CF team and see if there is any way we can try to revisit some of the lost functionality as future issues

Reply:
Here's what I'd like to propose as a solution to the problem with losing navigation history when navigating between screen reloads:

1. We'd have 2 menu items for migration (and also 2 rails routes in the plugin). That way we'd have vert. menu updates whenever we click on some of the migration menu items.
2. We'd go back to using `ConnectedRouter` and `Link`, which would give us back navigation history 
3. on the backend (or better yet classic-ui) side, we'd have an action, let's call it `vertical_menu_ping()`, which the client (v2v react in this case) would call anytime it would navigate between overview, plain id and migration mappings screens. This backend routine would:
- update vert. menu data in server session
- return back transformation data (jQuery code) which would re-style the vertical menu so that correct menu item is highlighted (Overview or Migration Mappings)

Questions, suggestions, ideas are welcome.